pkgs/development/tools/analysis/codeql/default.nix

```diff
@@ -12,15 +12,15 @@
 
 stdenv.mkDerivation rec {
   pname = "codeql";
-  version = "2.0.4";
+  version = "2.1.0";
 
   dontConfigure = true;
   dontBuild = true;
   dontStrip = true;
 
   src = fetchzip {
     url = "https://github.com/github/codeql-cli-binaries/releases/download/v${version}/codeql.zip";
```

Comment:
The release archives from github are not necessarily stable (e.g., they're generated dynamically and may change compression algorithm, modifying the `sha256`.  Can you use `fetchFromGitHub` instead?

Reply:
There are no binaries in the repo itself, so fetchFromGitHub is cannot be used here.

Reply:
Oh woah, I missed that this is an unfree package using github's release tarballs as a de-facto FTP server for distributing proprietary binaries.  First time I've seen something like that ...